scripts/lint-old-quo-usage.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
```

Comment:
```suggestion
set -euo pipefail
```

Reply:
@jakubgs I added this.

To handle the scenario when the `grep` command does not find any matches and prevent it from causing script termination, `|| true` has been added to `grep`. I'm not completely sure if this is the right way to handle this. Let me know if you have any suggestions.

Reply:
If failure to find instances via `grep` is expected then you can just drop the `-o pipefail` part.

Reply:
Oh, i see now, you're grepping the files directly. That's fine then, using `|| true` is a good solution.